### PR TITLE
New package: apache-maven-3.3.9

### PR DESCRIPTION
**Pull request by the-maldridge**
Proper source version of Apache Maven.

### CODE REVIEW

**Comment by the-maldridge** [comment]
bump?

File: srcpkgs/apache-maven/template [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Template file for 'apache-maven'
+pkgname=apache-maven
+version=3.3.9
+revision=1
+hostmakedepends="apache-ant openjdk"
+depends="virtual?java-runtime"
+noarch=yes
+short_desc="Software project management and comprehension tool"
+maintainer="Michael Aldridge <aldridge.mac@gmail.com>"
+license="Apache 2.0"
```
Comment (Vaelatern) [inline srcpkgs/apache-maven/template]: `Apache-2.0`

File: srcpkgs/apache-maven/template [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+	_installRoot=${DESTDIR}/usr/share/${pkgname}/
+	mkdir -p ${_installRoot}
+
+	for _dir in bin boot conf lib ; do
+		cp -r mvn-complete/${_dir} ${_installRoot}/${_dir}
+	done
+
+	mkdir -p ${DESTDIR}/usr/bin
+	for _app in mvn mvnDebug mvnyjp ; do
+		ln -s ${DESTDIR}/usr/share/${pkgname}/bin/${_app} ${DESTDIR}/usr/bin/${_app}
```
Comment (Vaelatern) [inline srcpkgs/apache-maven/template]: I believe these symlinks to be incorrect. ``${DESTDIR}/usr/share/${pkgname}/bin/${_app}` will not exist in an installed environment.

Comment (Vaelatern) [inline srcpkgs/apache-maven/template]: Do they need to be symlinked instead of copied?

Comment (Vaelatern) [inline srcpkgs/apache-maven/template]: Alternately might it be right to provide an `/etc/profile.d/` entry to append to `$PATH`?

Comment (the-maldridge) [inline srcpkgs/apache-maven/template]: I'll swap this out when I get a free moment.

**Pull request closed without merging**